v2/datastream-to-spanner/src/test/resources/DataStreamToSpanner100GbLT/spanner-schema-interleaving.sql

```diff
@@ -10,7 +10,7 @@ CREATE TABLE IF NOT EXISTS person2 (
      FK_ID2 INT64,
      ID INT64 NOT NULL,
 ) PRIMARY KEY(ID, FK_ID2),
-  INTERLEAVE IN PARENT person1 ON DELETE CASCADE;
```

Comment:
Please add tests for interleave in (without parent).

The existing tests for interleave in with parent should continue to exist